app/api/currentuser/current_user.go

```diff
@@ -43,6 +43,8 @@ func (srv *Service) SetRoutes(router chi.Router) {
 
 	router.Get("/current-user/full-dump", service.AppHandler(srv.getFullDump).ServeHTTP)
 	router.Get("/current-user/dump", service.AppHandler(srv.getDump).ServeHTTP)
+
+	router.Get("/current-user/team-by-item/{item_id}", service.AppHandler(srv.getTeamByItem).ServeHTTP)
```

Comment:
If you put it under `current-user`, I would do `/current-user/teams/by-item/{item_id}` (where `/current-user/teams/` would list all "my" teams (if the service would exist))

----

BTW, even if I agree that having it under `current-user/` is a good idea, I do not find that storing the code related to this very team/group-specific service in the "currentuser" package is so great. (same for the other services in "currentuser" actually). It would not allow sharing very team/group-specific helper functions in the api/groups package. What is your opinion?

Reply:
Changed the route. There is nothing to reuse from this service, but no problem, I moved it into /app/api/groups.